.ci/docker/common/install_cuda_aarch64.sh

```diff
@@ -17,6 +17,17 @@ function install_cusparselt_062 {
     rm -rf tmp_cusparselt
 }
 
+function install_cusparselt_063 {
+    # cuSparseLt license: https://docs.nvidia.com/cuda/cusparselt/license.html
+    mkdir tmp_cusparselt && pushd tmp_cusparselt
+    wget -q https://developer.download.nvidia.com/compute/cusparselt/redist/libcusparse_lt/linux-x86_64/libcusparse_lt-linux-x86_64-0.6.3.2-archive.tar.xz
```

Comment:
Missed this, should have been libcusparse_lt-linux-sbsa-0.6.3.2. This is causing error `/usr/local/cuda/lib64/libcusparseLt.so: error adding symbols: file in wrong format`